From 0e3b6450f6ab803442a809b6da41d5d5c5da650f Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Thu, 7 Jun 2012 15:36:16 +0100
Subject: [PATCH 02/12] Ensure empty compilerlibs/ directory is created by git.

This directory exists in the OCaml tarball, but is empty.  As a
result, git ignores it unless we put a dummy file in it.
---
 compilerlibs/.exists | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 compilerlibs/.exists

diff --git a/compilerlibs/.exists b/compilerlibs/.exists
new file mode 100644
index 0000000..e69de29
-- 
2.0.4

